.github/workflows/automated-release.yml

```diff
@@ -0,0 +1,116 @@
+name: Automated Release
+on:
+  release:
+    types: [published]
+jobs:
+  # release for linux
+  build-and-release-on-linux:
+    env:
+      TAG: ${{ github.ref_name }}
+      USER: merico-dev
+      REPO: stream
+      PLUGINDIR: .devstream
+      GOOS: linux
+      GOARCH: amd64
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🐧 This job is now running on a ${{ runner.os }}-${{ runner.arch }} server hosted by GitHub!"
+      - name: Checkout
```

Comment:
I think we should explicitly check out the tag. I don't know if it's possible, but what if I publish a release then immediately a new commit is generated by others? In that case, will it check out the latest commit instead of this tag?

Reply:
yes, i added a option for checkout the specified tag.